dev/com.ibm.ws.request.timing_fat/test-applications/jdbcTestPrj_3/src/com/ibm/ws/request/timing/TestJDBC.java

```diff
@@ -55,16 +55,16 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 
 
       System.out.println((new StringBuilder(" Session value is ")).append(s.getValue()).toString());
-
-       
+      
+      int returnValue = -9999999;
       for(int i = 11; i <= 15; i++) {
-          stmt.executeUpdate((new StringBuilder("insert into "+tableName+" values ('myHomeCity_ ")).append(i).append("', ").append(i).append(", 'myHomeCounty_").append(i).append("')").toString());
-          try {
-              Thread.sleep(5000);
-          } catch (Exception e) {
-              e.printStackTrace();
+          returnValue = stmt.executeUpdate((new StringBuilder("insert into "+tableName+" values ('myHomeCity_ ")).append(i).append("', ").append(i).append(", 'myHomeCounty_").append(i).append("')").toString());
+          // We will need to ensure that the executeUpdate completes before continuing
+          while (returnValue == -9999999) {
```

Comment:
Not sure, why this loop is needed here? 

If you want to make sure the executeUpdate completes, we can just check if the returnValue is populated with the number of row or 0, that would ensure the statement is completed.

Reply:
Not sure if the executeUpdate run asynchronously here but I will add the thread.sleeps back (like the original) but shorten the time to sleep for as each update normally takes a few milliseconds to complete. I will also add some checking to the return value of the executeUpdate to ensure the update is successful.